contents/docs/libraries/js/index.mdx

```diff
@@ -514,6 +516,7 @@ Some of the most relevant options are:
 | `capture_pageleave`<br/><br/>**Type:** Boolean<br/>**Default:** `true`                                                                                                                     | Determines if PostHog should automatically capture pageleave events.                                                            |
 | `cross_subdomain_cookie`<br/><br/>**Type:** Boolean<br/>**Default:** `true`                                                                                                                     | Determines if cookie should be set on the top level domain (example.com). If PostHog-js is loaded on a subdomain (test.example.com), _and_ `cross_subdomain_cookie` is set to false, it'll set the cookie on the subdomain only (test.example.com).                                                           |
 | `disable_persistence`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                                 | Disable persisting user data across pages. This will disable cookies, session storage and local storage.                       |
+| `disable_surveys`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                           | Determines if surveys script should load, and whether surveys should show up for users               |
```

Comment:
```suggestion
| `disable_surveys`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                           | Determines if the surveys script should load which controls whether they show up for users               |
```

Reply:
Oh interesting, they can be separate, will edit this to make this clearer